src/main/webapp/app/entities/search/search.reducer.ts

```diff
@@ -104,7 +104,7 @@ export const searchChange: (viewId: string, text: string) => void = (viewId, tex
   });
 };
 
-export const doSearch: (viewId: string, text: string) => void = (viewId, text) => dispatch => {
+export const doSearch: (viewId: string, text: any) => void = (viewId, text) => dispatch => {
```

Comment:
why text is not string?

Reply:
because we convert text to query DTO and query DTO is object